.github/ISSUE_TEMPLATE/report_a_bug.yml

```diff
@@ -0,0 +1,144 @@
+name: 🐛 Report a bug
+description: Have you found a bug or issue? Create a bug report for OpenFGA
+labels: [ ":beetle: bug" ]
```

Comment:
This is a proposal for a new label to be created before this PR gets merged.

Reply:
Removed in [`8b59dc8` (#1262)](https://github.com/openfga/openfga/pull/1262/commits/8b59dc8d3ff120b7d0e8cb521b4101cfc5e90df8).